tests/data/stac/issue_copernicus_global_mosaics/stac.dataspace.copernicus.eu/v1/index.html.json

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
is that `.html.json` extension intentional?

Reply:
It is what wget outputted for me. Changed the name to index.json